File: Darus-Nicolae NAGY/lab4-azure/src/main/kotlin/Student.kt
import com.microsoft.azure.storage.table.TableServiceEntity

class Student(partitionKey: String?,
              rowKey: String?) : TableServiceEntity(partitionKey, rowKey) {
    constructor(partitionKey: String?,
                rowKey: String?, name: String?, email: String?, year: Int?) : this(partitionKey, rowKey) {
        this.email = email
        this.name = name
        this.year = year
    }

    constructor() : this(null, null)

    var name: String? = null
    var email: String? = null
    var year: Int? = null

    override fun toString(): String {
        return "$partitionKey, $rowKey, $name, $email, $year"
    }
}

File: Darus-Nicolae NAGY/lab2/Lab2App/Lab2App/Model/Student.cs
namespace Lab2App.Model
{
    public class Student
    {
        public Student(long id, string name, string faculty, int year)
        {
            Id = id;
            Name = name;
            Faculty = faculty;
            Year = year;
        }
        public long Id { get; set; }
        public string Name { get; set; }
        public string Faculty { get; set; }
        public int Year { get; set; }
    }
}